kbase/narrative
# DATAUP-266 staging refresh

# Description of PR purpose/changes

As reported in DATAUP-266, when a user finishes uploading a file, the file viewer doesn't refresh automatically anymore. This restores that functionality. I also noticed some browser slowdown / lockup when uploading a lot of small files at one time, which trigger a refresh (and a `list` call to the staging service) when each file ends. If the files are small enough, or the user's upload speed is fast enough, this might cause problems. So the refresh is only allowed to run up to once per second.

A side effect of this is setting the local `config.json` to use the `dev` environment. This is currently needed to be able to run tests against CI, and will have no effect on either the built image or deployment.

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X

# Testing Instructions
* Details for how to test the PR: 
Just upload a file, and it should refresh and appear right away.

# Dev Checklist:

- [] My changes generate no new warnings
